portal/src/app/components/home/components/experiment/components/table-options/constants/table-options.const.ts

```diff
@@ -1,20 +1,28 @@
 import { EXPERIMENT_TABLE_EN } from '../../experiment-table/translate/en';
 import { Environment } from "../../../../../../../../environments/environment";
 import { DashBoardPrefixLink } from "../../../../../../../shared/constants/dashboard";
+import { IResult, ITestRunResultData } from '../../../../../../../shared/models/test-run-result.interface';
+
+type SelectedColumnsDefaultData = {
+  label: keyof ITestRunResultData | `results.${keyof IResult}`;
```

Comment:
`${results keyof IResult}` not working?

Reply:
no, because the label should be one of the followings:
- A property of `ITestRunResultData`.
- A property inside results, so that the key will have to be `results.<property_of_IResult>`.